### PR TITLE
Fix events on image icons

### LINKED ISSUE #73: Fix some Icons
List of icons to be replaced:

Screenshot 2023-04-26 at 14 15 42  At "Informação pessoal" Page -> "Adicionar Nacionalidade"
Screenshot 2023-04-26 at 14 18 12 At "Telemóvel" in "Informação pessoal" Page

### LINKED ISSUE #81: Popup da página informação pessoal não abrem
Problems with "a" tags on clicks functions

### PR DESCRIPTION
**Pull request by ttoino**
Closes #81.
Closes #73.

Now image icons are not replaced, but hidden, and events are delegated to the original element.

### CODE REVIEW

**Review by LuisDuarte1 — CHANGES_REQUESTED** [review]
It seems that icons are duplicated on popups, but only on the first load of a popup for some reason, you can replicate on your personal info edit page

**Comment by LuisDuarte1** [comment]
image

Also replaced the notification message icon on sigarra, because it was broken and not everyone has this

**Comment by ttoino** [comment]
> 

I also noticed this but thought it was because of the hotreload, I'll try to fix it.

**Comment by ttoino** [comment]
> 
> 
> It seems that icons are duplicated on popups, but only on the first load of a popup for some reason, you can replicate on your personal info edit page

It seems to be fixed, turns out I misunderstood how `:scope` works :upside_down_face:

File: content-scripts/src/modules/icons/index.js [hunk 1]
```diff
@@ -16,31 +17,49 @@ const replaceImages = () => {
...
         span.style.fontSize = `${size}px`;
+        span.classList.add("se-icon");
+        span.classList.remove("se-hidden-icon");
     };
 
     new MutationObserver((ms) =>
         ms.forEach((m) => {
-            if (m.type == "attributes") handleImage(m.target);
-            else m.target.querySelectorAll("img").forEach(handleImage);
+            console.log(m);
```
Comment (LuisDuarte1) [inline content-scripts/src/modules/icons/index.js:55]: Maybe remove this `console.log` if it's not necessary

**Review by LuisDuarte1 — CHANGES_REQUESTED** [review]
LGTM, just look to the comment below :)

File: content-scripts/src/modules/icons/index.js (hunk 1 above)
Comment (ttoino) [reply content-scripts/src/modules/icons/index.js:55]: You're right, forgot to remove that

**Review by LuisDuarte1 — APPROVED** [review]
LGTM 🚀

**Pull request merged**